lib/utilities/paginator.rb

```diff
@@ -0,0 +1,85 @@
+module Utilities
+  module Paginator
+    def items_to_display(item:, items_per_page:)
+      @items_per_page = items_per_page || 5
+      @current_page = params[:page] ? params[:page].to_i : 1
+      @number_of_items = item.all.count
+      @last_page = (@number_of_items.to_f / @items_per_page).ceil
+      item.offset(items_per_page * (@current_page - 1))
+          .limit(items_per_page)
+    end
+
+    def paginator(max_page_blocks: 7)
+      render partial: 'shared/paginator',
+             locals: { blocks: pages_to_display(@current_page,
+                                                max_page_blocks,
+                                                @last_page),
+                       current_page: @current_page,
+                       previous_status: previous_status,
+                       next_status: next_status,
+                       last_page: @last_page }
+    end
+
+    private
+
+    def previous_status
```

Comment:
This method could be more semantic. It returns the css class name for the "previous" link